### PR TITLE
fix(transformer): Event getters now use property name not event name

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: modules/angular2/src/transform/bind_generator/generator.dart [hunk 1]
```diff
@@ -105,8 +105,8 @@ Map<String, String> _createEventsMap(NgDeps ngDeps) {
     visitor.bindConfig.forEach((String config) {
       // See comments for `Directive` in annotations_impl/annotations.ts for
       // details on how `events` is specified.
-      var parts = config.split(':').map((p) => p.trim()).toList();
-      bindMap[parts[0]] = parts.length > 1 ? parts[1] : parts[0];
+      var eventName = config.split(':').first.trim();
+      bindMap[eventName] = eventName;
```
Comment (yjbanov) [inline modules/angular2/src/transform/bind_generator/generator.dart]: Does it still need to be a map then?

File: modules/angular2/src/transform/bind_generator/generator.dart [hunk 2]
```diff
@@ -105,8 +105,8 @@ Map<String, String> _createEventsMap(NgDeps ngDeps) {
     visitor.bindConfig.forEach((String config) {
       // See comments for `Directive` in annotations_impl/annotations.ts for
       // details on how `events` is specified.
-      var parts = config.split(':').map((p) => p.trim()).toList();
-      bindMap[parts[0]] = parts.length > 1 ? parts[1] : parts[0];
+      var eventName = config.split(':').first.trim();
```
Comment (yjbanov) [inline modules/angular2/src/transform/bind_generator/generator.dart]: Let's rename the variable to `eventStreamGetter`

File: modules/angular2/src/transform/bind_generator/generator.dart (hunk 1 above)
Comment (jakemac53) [inline modules/angular2/src/transform/bind_generator/generator.dart]: nope, it doesn't

File: modules/angular2/src/transform/bind_generator/generator.dart (hunk 2 above)
Comment (jakemac53) [inline modules/angular2/src/transform/bind_generator/generator.dart]: removed the local variable entirely

**Comment by jakemac53** [comment]
ok - all updated

**Pull request merged**